### PR TITLE
fixed: airiti getDocId function

### PR DESCRIPTION
**Pull request by cesaryuan**
fixed airiti cannot getDocId with lowercase url paramters

### CODE REVIEW

File: Airiti.js [hunk 1]
```diff
@@ -20,13 +20,14 @@ function detectWeb(doc, url) {
...
 function getDocId(url) {
-	var m = url.match(/\bDocID=([^&#]+)/);
+	var m = url.match(/\bDocID=([^&#]+)/i);
 	if (!m) return;
 	return m[1];
 }
 
 async function doWeb(doc, url) {
 	var docID = getDocId(url);
+	Z.debug(`doWeb: ${docID}`);
```
Comment (AbeJellinek) [inline Airiti.js:30]: This should be removed

File: Airiti.js [hunk 2]
```diff
@@ -54,6 +55,7 @@ async function scrape(docIDs, itemDoneHandler) {
 	await requestDocument('https://www.airitilibrary.com/Publication/MetaExport?' + tokenParams);
 
 	var bibTeXUrl = buildQuery(docIDs);
+	Z.debug('bibTeXUrl: ' + bibTeXUrl);
```
Comment (AbeJellinek) [inline Airiti.js:58]: And this

**Review by AbeJellinek — CHANGES_REQUESTED** [review]
Looks good, just remove the debug calls!

File: Airiti.js (hunk 1 above)
Comment (cesaryuan) [reply Airiti.js]: Thank you. I have removed the calls.

**Comment by AbeJellinek** [comment]
Thanks!

**Pull request merged**